### PR TITLE
v1.51 Mobile Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/deploy/mobile-app-changelog.md [hunk 1]
```diff
@@ -1,12 +1,49 @@
...
+ - **Upgrade to server version v6.3.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 5.37 has ended and upgrading to server ESR v6.3.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Breaking Changes
+ - Apps calls are now separated between submit, form, refresh and lookup calls.
+
+### Improvements
+ - Users are now taken directly to the SSO login if only one SSO login method is enabled.
+ - Calls are now enabled on mobile if the server has Calls enabled.
```
Comment (amyblais) [inline source/deploy/mobile-app-changelog.md:25]: Do we want to mention Calls already in the changelog, or do we remove the Calls items for now?

File: source/deploy/mobile-app-changelog.md [hunk 2]
```diff
@@ -1,12 +1,49 @@
...
+- [1.49.1 Release](#id11)
+- [1.49.0 Release](#id14)
+- [1.48.2 Release](#id20)
+
+## 1.51.0 Release
+- Release Date: April 16, 2022
+- Server Versions Supported: Server v6.3.0+ is required. Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device.
+
+### Compatibility
+ - **Upgrade to server version v6.3.0 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/upgrade/extended-support-release.html) (ESR) 5.37 has ended and upgrading to server ESR v6.3.0 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
```
Comment (enahum) [inline source/deploy/mobile-app-changelog.md:16]: this is only in the release notes but not enforced, right ?

File: source/deploy/mobile-app-changelog.md [hunk 3]
```diff
@@ -1,12 +1,49 @@
...
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).	
+ - iPhone 5s devices and later with iOS 12.1+ is required.
+
+### Breaking Changes
+ - Apps calls are now separated between submit, form, refresh and lookup calls.
```
Comment (enahum) [inline source/deploy/mobile-app-changelog.md:21]: @catalintomai is this ok?

File: source/deploy/mobile-app-changelog.md (hunk 1 above)
Comment (enahum) [reply source/deploy/mobile-app-changelog.md:25]: @itao ☝️

File: source/deploy/mobile-app-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/deploy/mobile-app-changelog.md:16]: That's correct.

Comment (mickmister) [reply source/deploy/mobile-app-changelog.md:16]: @enahum Is it guaranteed that there are crashes here? A server I'm maintaining is currently behind ESR at 5.38

File: source/deploy/mobile-app-changelog.md (hunk 3 above)
Comment (mickmister) [reply source/deploy/mobile-app-changelog.md:21]:
```suggestion
 - The Apps Framework protocol for binding/form submissions has changed, by separating the single `call` into separate `submit`, `form`, `refresh` and `lookup` calls.
```

I think this wording works well. It's unfortunate that there is terminology overlap with the Calls feature.

File: source/deploy/mobile-app-changelog.md [hunk 4]
```diff
@@ -1,12 +1,49 @@
...
+#### All apps
+ - Fixed an issue with Calls banner locations for earlier iPhone models.
+ - Fixed an issue where users were not able to scroll down to view custom themes.
+ - Fixed a bug that resulted in dropping the Call state when switching to another app and returning.
+
+### Known Issues
+ - Channel sidebar disappears sometimes when the channel categories are not fetched from the server.
+ - Posts sometimes get stuck behind the post textbox on iPad.
+ - Various known issues with Collapsed Reply Threads (Beta) feature:
+   - New messages banner should only count root posts.
```
Comment (esethna) [inline source/deploy/mobile-app-changelog.md:45]: Wasn't this fixed? https://mattermost.atlassian.net/browse/MM-37598

File: source/deploy/mobile-app-changelog.md [hunk 5]
```diff
@@ -1,12 +1,49 @@
...
+   - Threads item unread state (bolding) does not persist when deleting documents and data.
```
Comment (esethna) [inline source/deploy/mobile-app-changelog.md:46]: This one was closed: https://mattermost.atlassian.net/browse/MM-37794

**Review by esethna — APPROVED** [review]

File: source/deploy/mobile-app-changelog.md (hunk 1 above)
Comment (cpoile) [reply source/deploy/mobile-app-changelog.md:25]: I checked -- we're okay to release Calls-related changelog entries.

**Review by cpoile — APPROVED** [review]
👍

**Review by catalintomai — APPROVED** [review]

**Review by larkox — APPROVED** [review]

**Pull request merged**